js/src/views/Signer/components/SignRequest/signRequest.css

```diff
@@ -27,8 +27,18 @@
   min-height: $pendingHeight;
 }
 
+.signData {
```

Comment:
Might be good to introduce maximal height and overflow for very long ascii texts:
```
overflow: auto;
max-height: 6em;
```